# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/xpcom/AccEvents.conf
# Commit: 786805a2cdcc
# Full Hash: 786805a2cdcc4cdab632a955dace417062d0087e
# Author: James Teh <jteh@mozilla.com>
# Date: 2023-11-30 04:12:04
# Regressor Bug: 1862802
# File Overlap Count: 1
# Description:
#   Bug 1862802 part 5: Remove virtual cursor change event. r=eeejay
#   
#   Depends on D192645
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D192646
# ==============================================================================

diff -r effe12f2120a -r 786805a2cdcc accessible/xpcom/AccEvents.conf
--- a/accessible/xpcom/AccEvents.conf	Tue Nov 28 22:03:20 2023 +0000
+++ b/accessible/xpcom/AccEvents.conf	Tue Nov 28 22:03:21 2023 +0000
@@ -15,7 +15,6 @@
     'CaretMoveEvent',
     'ObjectAttributeChangedEvent',
     'TableChangeEvent',
-    'VirtualCursorChangeEvent',
     'ScrollingEvent',
     'AnnouncementEvent'
   ]
